contracts/Import.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.9;
+
+import '@axelar-network/axelar-cgp-solidity/contracts/gas-service/AxelarGasService.sol';
```

Comment:
Did you check if the same bytecode is derived after compilation? We want to derive the same address on all chains

Reply:
If it gives a different address, then maybe you can try that npm package to compile imported contracts in case that behaves better

Reply:
It seems to produce the same address on different chains yes, but this is only a temporary solution until we republish the cgp npm module to include the artifacts folder. Let me know if you prefer that I use the hardhat-dependency-compiler npm package in the meantime

Reply:
^ as per our discussion bytecode is not same when compared to our previous deployment method so I updated the PR to manually include required artifacts